ToDoListApp/View/ViewController.swift

```diff
@@ -44,6 +64,7 @@ extension ViewController: UITableViewDataSource {
         let task = tasks[indexPath.row]
```

Comment:
this can cause a crash for index out of range 
```suggestion
if indexPath.count > indexPath.row {
        let task = tasks[indexPath.row]
        
```